### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by nicksolomon**

### CODE REVIEW

File: chapter4.md [hunk 1]
```diff
@@ -863,7 +863,7 @@ np.median(x)
 The baseball data is available as a 2D Numpy array with 3 columns (height, weight, age) and 1015 rows. The name of this Numpy array is `np_baseball`. After restructuring the data, however, you notice that some height values are abnormally high. Follow the instructions and discover which summary statistic is best suited if you're dealing with so-called _outliers_.
 
 *** =instructions
-- Create Numpy array `np_height`, that is equal to first column of `np_baseball`.
+- Create Numpy array `np_height` that is equal to first column of `np_baseball`.
```
Comment (hugobowne) [inline chapter4.md]: Numpy here is inconsistent with the rest

File: chapter4.md [hunk 2]
```diff
@@ -361,21 +361,21 @@ success_msg("Wow! It appears that only 11 of the more than 1000 baseball players
...
 
 --- type:MultipleChoiceExercise lang:python xp:50 skills:2 key:3662ff6637
-## Numpy Side Effects
+## NumPy Side Effects
 
-As Filip explained before, Numpy is great for doing vector arithmetic. If you compare its functionality with regular Python lists, however, some things have changed.
+As Filip explained before, `numpy` is great for doing vector arithmetic. If you compare its functionality with regular Python lists, however, some things have changed.
 
-First of all, Numpy arrays cannot contain elements with different types. If you try to build such a list, some of the elments' types are changed to end up with a homogenous list. This is known as _type coercion_.
+First of all, `numpy` arrays cannot contain elements with different types. If you try to build such a list, some of the elments' types are changed to end up with a homogenous list. This is known as _type coercion_.
```
Comment (yrdatacamp) [inline chapter4.md]: A couple of typos here: elments --> elements and homogenous --> homogenous.

**Comment by nicksolomon** [comment]
I wen through the all the exercise typos and fixed them.

The one slightly larger change I made was to clarify the units in an exercise in the NumPy chapter. Users were confused because it seemed the columns were backwards. It turns out that reasonable values for height in inches are similar to reasonable values for weight in kilograms, while normal values of weight in pounds are close to normal values of height in centimeters.

**Comment by yrdatacamp** [comment]
@hugobowne Having standardized the NumPy styling, there's now a discrepancy between video titles (NumPy) and slide titles (Numpy). I'll take care of that and propose we hold off on merging until then (~ Tuesday). Great work @nicksolomon.

**Comment by yrdatacamp** [comment]
@hugobowne The relevant videos have been edited here: https://github.com/datacamp/courses-intro-to-python/pull/17. This is good to merge!

**Pull request merged**